tests/test_qasm_examples.py

```diff
@@ -1,17 +1,86 @@
 import os
 import re
 import pytest
+import numpy as np
 
 from qdata import parser
 
+
+def split_at_semicol(qasm_lines):
+    """Helper function to split lines at semicolons"""
+    form_lines = []
+    for line_1 in qasm_lines:
+        split_lines = line_1.split(";")
+        if len(split_lines) == 1:
+            form_lines.append(split_lines[0])
+        elif len(split_lines) == 2 and split_lines[1] == "":
+            form_lines.append(split_lines[0] + ";")
```

Comment:
How come this elif can occur? This is the case where the line looks like `"something;"`?

Reply:
Yes, it should happen every time there's a correct statement that ends with a `;` (e.g. `something;`). But looking at it now it seems like it should be covered by the following `else`. :thinking:

Reply:
Yes, that appears to be true